### PR TITLE
Smoother cryotube health bars + icon cleanup

### PR DESCRIPTION
**Pull request by SECBATON-GRIFFON**
[tested][sprites]

## What this does
changes the preset icons for the progress bars to a one masked out with Blend() and SwapColor() procs, also removes a few unused icons in this file

## Why it's good
looks much smoother, icon cleanup

## Changelog
:cl:
 * tweak: Cryotube health bars now move much smoother.

### CODE REVIEW

**Comment by PrimeDSS13** [comment]
How does it look in game?

**Comment by Exxion** [comment]
Really do not like putting the legendarily slow icon procs in something called this often
It's probably possible to do something similar with `blend_mode` and `pixel_x`

**Comment by SECBATON-GRIFFON** [comment]
> Really do not like putting the legendarily slow icon procs in something called this often

would the mask filter() be faster or not

**Comment by Exxion** [comment]
Surely yes

**Comment by SECBATON-GRIFFON** [comment]
> It's probably possible to do something similar with `blend_mode` and `pixel_x`

it seems either BLEND_SUBTRACT or BLEND_INSET_OVERLAY is the way to go with blend_mode right? is there a way to make it only target the image below it and not the whole object icon? looking at this right now: https://www.byond.com/docs/ref/#/atom/var/blend_mode

**Comment by Exxion** [comment]
I think this is one of the advanced uses of `KEEP_TOGETHER`, but I'm not 100% sure how it's done

**Comment by SECBATON-GRIFFON** [comment]
okay, i managed to set up BLEND_INSET_OVERLAY but the mask itself seems to be masked by the actual object, not the other way around. it looks almost equivalent but the empty space is black. is there a way to remove this masking part itself to make its alpha 0 but also remove whats underneath?

File: code/game/machinery/cryo.dm [hunk 1]
```diff
@@ -405,32 +400,29 @@ var/global/list/cryo_health_indicator = list(	"full" = image("icon" = 'icons/obj
...
+								if(-100 to 0)
+									healthoverlay = cryo_health_indicator["crit"]
 								else //Shouldn't ever happen. I really hope it doesn't ever happen.
-									overlays += cryo_health_indicator["dead"]
+									healthoverlay = cryo_health_indicator["dead"]
+							var/image/mask = cryo_health_indicator["mask"]
+							healthoverlay.appearance_flags = KEEP_TOGETHER
+							mask.blend_mode = BLEND_INSET_OVERLAY
+							mask.pixel_x = max(3,3+(14*abs(occupant.health / occupant.maxHealth)))
+							mask.color = "#000"
```
Comment (SECBATON-GRIFFON) [inline code/game/machinery/cryo.dm:422]: okay, i managed to set up BLEND_INSET_OVERLAY but the mask itself seems to be masked by the actual object, not the other way around. it looks almost equivalent but the empty space is black. is there a way to remove this masking part itself to make its alpha 0 but also remove whats underneath?

**Comment by Kurfursten** [comment]
is this as good as it will ever be?

**Comment by SECBATON-GRIFFON** [comment]
> is this as good as it will ever be?

probably, can't find any good documentation on how blend masking works. ill make another PR if i discover anything.

**Pull request merged**